projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/review-submit/review-submit.component.spec.ts

```diff
@@ -85,8 +85,14 @@ describe('ReviewSubmitComponent', () => {
       loadSupportedDeliveryModes: createSpy()
     };
     mockCartService = {
-      activeCart$: new BehaviorSubject(null),
-      entries$: new BehaviorSubject(null)
+      cart$: new BehaviorSubject(null),
+      entries$: new BehaviorSubject(null),
+      getActiveCart(): BehaviorSubject<null> {
```

Comment:
Can we add a more specific type instead of `null` in `BehaviorSubject<null>`?

Reply:
Applies to `getEntries()` as well

Reply:
Not quite sure what type it would be, would it be BehaviorSubject<Cart> for getActiveCart()?

Reply:
Just use the same type as in the real service.